docs/development_guide.md

```diff
@@ -167,9 +167,24 @@ TODO:
 
 ## Helm chart
 
-The Open Service Mesh control plane chart is located in the
+The Open Service Mesh control plane charts are located in the
```

Comment:
The whole directory is called a "chart," so the way it was before makes more sense.

Reply:
Updating.